src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2121,263 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The definition for subscribing emailId's to query or schedule IDs for a given alertType
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+    properties:
+      assetId:
+        type: string
+        description: The query or schedule ID that the alert is created for and subscribed to.
+      alertType:
+        type: string
+        description: The alert type is one of three values (start, success, failure). This determines the state of query execution that triggers a user to receive an alert.
```

Comment:
Describe the meaning of the field before listing its potential values. "The state of query execution that triggers the alert for its subscribers. Potential values are `start`, `success`, and `failure`."